odoo/addons/base/models/res_currency.py

```diff
@@ -139,18 +139,19 @@ def _compute_is_current_company_currency(self):
             currency.is_current_company_currency = self.env.company.root_id.currency_id == currency
 
     @api.depends('rate_ids.rate')
+    @api.depends_context('to_currency', 'date', 'company')
     def _compute_current_rate(self):
         date = self._context.get('date') or fields.Date.context_today(self)
         company = self.env['res.company'].browse(self._context.get('company_id')) or self.env.company
```

Comment:
Is the context key `company_id` still relevant? If so, it should be in the `depends_context()` decorator, shouldn't it? (In the past, we introduced `env.company` specifically to unify a bit the use of such keys.)

Reply:
I'm adding it just to avoid breaking things.
The last diff to this lines actually comes from the commit unifying the API https://github.com/odoo/odoo/commit/f5dfe4727c086c7577da2aef12c2d3548ba938a3

Not sure if missed or one purpose :/